### ISSUE TITLE
BUG/ENHANCEMENT - Caching Responses from POST requests

### ISSUE THREAD

**Issue by sanchezcarlosjr** [karpetrosyan/hishel#158]
Hello Hishel team,

The issue at https://github.com/karpetrosyan/hishel/issues/36 mentions that we can specify any method we want to cache, as detailated in the [documentation](https://hishel.com/advanced/controllers/#cachable-http-methods). However, when I used Hishel version 0.0.21, I encountered the following error:
```
RuntimeError: Hishel does not support the HTTP method `POST`. Please use either `GET` or `HEAD`.
```
This occurred with the given controller setup:
```
controller = hishel.Controller(allow_stale=True, cacheable_methods=["GET", "POST", "PUT"])
```

Is there something that I'm missing?

Thank you very much!

**Comment by karpetrosyan** [comment]
Hi @sanchezcarlosjr
Thank you for bringing this to our attention!

> Is there something that I'm missing?

It's just a documentation issue. I believe the only methods that can be cached are GET and HEAD, so we simply need to change PUT in the documentation to HEAD.
Would you like to submit a pull request to fix this?

> We should add a SimpleController that does not follow RFC 9111, but just works off of methods and status codes. I would have a use for that class myself!

It's an interesting idea, but I think we should discuss it in a separate issue.

**Comment by umnovI** [comment]
Support for POST method is now available in 0.0.24.
You can see here examples of how to enable it #183
